src/vaadin-message.js

```diff
@@ -151,7 +153,6 @@ class MessageElement extends ElementMixin(ThemableMixin(PolymerElement)) {
 
   ready() {
     super.ready();
-    this.setAttribute('aria-live', 'polite');
     this.setAttribute('role', 'listitem');
```

Comment:
I think this should be moved to src/vaadin-message-list.js#87. It would mean that the message-list puts role="listitem" on message. If someone would initialize a message standalone, then there wouldn't be dangling role="listitem", without a parent with role="log".